client/src/stores/ShippingStore.js

```diff
@@ -0,0 +1,44 @@
+import { makeObservable, observable, action, computed } from 'mobx';
+
+class ShippingStore {
```

Comment:
Will the `ShippingStore` still be necessary when the shipping API is available?

Reply:
I was wondering that we shouldn't call the API every time we render components in the page. In addition, is it better if we call the APIs when the user logs in and initialize the stores?

Reply:
> I was wondering that we shouldn't call the API every time we render components in the page. In addition, is it better if we call the APIs when the user logs in and initialize all the stores?

Those two concerns are solved by `react-query` which we are already using in the invenroty and admin pages.

Reply:
I will look for more in details on the `shippingStore ` and `react-query`

Reply:
@gordonpn  I will refactor once I have created the shipping APIs

Reply:
@DavidLiang01 
> I will refactor once I have created the shipping APIs

Sounds good